packages/react/src/primitives/SearchField/useSearchField.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+
+import { isFunction } from '../shared/utils';
+import { SearchFieldProps } from '../types';
+import { useComposeRefsCallback } from '../../hooks/useComposeRefsCallback';
+
+const ESCAPE_KEY = 'Escape';
+const ENTER_KEY = 'Enter';
+const DEFAULT_KEYS = [ESCAPE_KEY, ENTER_KEY];
```

Comment:
suggestion: convert it into a `Set`. Lookups are faster than an array (`Set.prototype.has` v/s `Array.prototype.includes`)

```suggestion
const DEFAULT_KEYS = new Set([ESCAPE_KEY, ENTER_KEY]);
```